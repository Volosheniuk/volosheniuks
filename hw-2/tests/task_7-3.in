1213sSneugfq87g3b4fu43p8t 

