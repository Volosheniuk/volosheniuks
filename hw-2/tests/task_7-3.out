YOURS STRING:1213sSneugfq87g3b4fu43p8t
0
String after rule2:11221133ssSSnneeuuggffqq8877gg33bb44ffuu4433pp88tt
